tests/phpunit/tests/modules.php

```diff
@@ -0,0 +1,429 @@
+<?php
+/**
+ * @group modules
+ * @covers ::wp_register_module
+ * @covers ::wp_enqueue_module
+ * @covers ::wp_dequeue_module
+ */
+class Tests_Modules extends WP_UnitTestCase {
```

Comment:
Maybe split the tests here into two files, one for the functions and one for the class methods. There's definitely a lack of `@covers` annotations for all the methods being tested here, such as for `print_module_preloads` or `print_import_map`.

Reply:
Split and added `@covers` for the print methods in https://github.com/WordPress/wordpress-develop/pull/5818/commits/76e87268154ea5938c111bc12478bd37d8c45bc0.